modules/integration/src/test/java/org/apache/fluo/integration/impl/CollisionIT.java

```diff
@@ -81,27 +77,22 @@ public void load(TransactionBase tx, Context context) throws Exception {
     }
   }
 
-  public static class TotalObserver extends AbstractObserver {
-
+  public static class CollisionObserverProvider implements ObserverProvider {
     @Override
-    public Observer.ObservedColumn getObservedColumn() {
-      return new Observer.ObservedColumn(STAT_CHANGED, NotificationType.WEAK);
-    }
-
-    @Override
-    public void process(TransactionBase tx, Bytes rowBytes, Column col) throws Exception {
-      String row = rowBytes.toString();
-      int total = Integer.parseInt(tx.gets(row, STAT_TOTAL));
-      int processed = TestUtil.getOrDefault(tx, row, STAT_PROCESSED, 0);
-
-      tx.set(row, STAT_PROCESSED, total + "");
-      TestUtil.increment(tx, "all", STAT_TOTAL, total - processed);
+    public void provide(Registry consumer, Context ctx) {
```

Comment:
I switched everything to use `or` as a short for observer registry.  Its probably too short though.  Still trying to think of a shorter variable name for this.  `observerRegistry` is too long.  May just go with `registry` as you suggested.